ert3/engine/_record.py

```diff
@@ -37,4 +37,5 @@ def sample_record(
         record_name=record_name,
         ensemble_record=ensrecord,
         experiment_name=experiment_name,
+        record_class="parameter",
```

Comment:
I think we should have the record class on the record itself. So pass it to `EnsembleRecord` a few lines up.